app/Models/User.php

```diff
@@ -17,9 +17,14 @@ class User extends Authenticatable
      * @var array<int, string>
      */
     protected $fillable = [
-        'name',
+        'first_name',
+        'last_name',
+        'address',
```

Comment:
Adress not in E/R diagram

Reply:
Unused address has been removed from user model